src/test/java/org/sagebionetworks/bridge/services/ParticipantServiceTest.java

```diff
@@ -2457,6 +2457,72 @@ public void getParticipantRoster() throws JsonProcessingException {
         verify(sqsClient).sendMessage(queueUrl, requestJson);
     }
 
+    @Test
+    public void updateParticipantNoteSuccessfulAsAdmin() {
```

Comment:
Instead of starting with Null, begin with a value to make sure it's not cleared and it's overwritten.